lib/web_ui/lib/src/engine/compositor/embedded_views.dart

```diff
@@ -369,6 +370,8 @@ class HtmlViewEmbedder {
       if (_overlays[viewId] != null) {
         final Overlay overlay = _overlays[viewId]!;
         overlay.surface.htmlElement?.remove();
+        overlay.surface.htmlElement = null;
+        overlay.skSurface?.dispose();
```

Comment:
Should we also null it out?

Reply:
I can't because `skSurface` is `final`. It should be garbage collected anyways after `overlay`